### DISCUSSION TITLE
How do you use JavaToCSharp?

### DISCUSSION THREAD

**Discussion by paulirwin** [paulirwin/JavaToCSharp#87] [Polls]
JavaToCSharp is available as a GUI, CLI, and core library on NuGet. We'd like to know how you use JavaToCSharp to know where to focus our efforts and compatibility!

**Comment by nzdev** [comment]
What would be helpful is a way to mix hand written code and automatically translated code, such that it would be possible to mostly automate java to C# conversion and mix in handwritten C# code.

Take in a C# partial class that has annotations.
e.g. [ImplementsJavaClass("org.myjava.myjavaclass")], [IgnoreJavaClass("org.myjava.myjavaclass")] , [ImplementsJavaMethod("myMethod")] , [IgnoreJavaMethod("myMethod")], [IgnoreJavaImport("java.io'")]  etc.
